software/base/src/test/java/brooklyn/entity/AbstractEc2LiveTest.java

```diff
@@ -75,10 +75,11 @@ public void setUp() throws Exception {
         super.setUp();
     }
 
+    // Image ids for Debian: https://wiki.debian.org/Cloud/AmazonEC2Image/Squeeze
     @Test(groups = {"Live"})
     public void test_Debian_6() throws Exception {
         // release codename "squeeze"
-        runTest(ImmutableMap.of("imageId", "us-east-1/ami-0740476e", "loginUser", "admin", "hardwareId", SMALL_HARDWARE_ID));
+        runTest(ImmutableMap.of("imageId", "us-east-1/ami-5e12dc36", "loginUser", "admin", "hardwareId", SMALL_HARDWARE_ID));
```

Comment:
i was thinking the machine ID -- but if it's localised to this one method no need i guess.  really just curious why this one and centos5 changed.  are the old images deprecated?